packages/better-auth/package.json

```diff
@@ -9,7 +9,7 @@
     "directory": "packages/better-auth"
   },
   "scripts": {
-    "build": "cross-env NODE_OPTIONS=--max-old-space-size=8000 tsup --clean --dts --minify",
```

Comment:
How about keeping the original build command, and introducing a new comamnd "dev:build" instead?

Reply:
It’s a significant issue when you encounter an error in a project, check the source code, and realize it’s fully minified. While compressing code is suitable for CDN usage, it’s unnecessary for an npm package. Modern frameworks already handle compression during the build process.

(I should be able to put console.log in the source code immediately.)